Source/Core/InputCommon/InputProfile.h

```diff
@@ -1,53 +1,30 @@
-// Copyright 2018 Dolphin Emulator Project
+// Copyright 2019 Dolphin Emulator Project
```

Comment:
This doesn't need to be changed when modifications are made to the file. It's intended to establish a creation date.